src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="surname">
```

Comment:
no empty lines between parent and child elements
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
      <fieldset class="fieldset">
        <legend>Personal information</legend>

        <label class="form-field" for="surname">
```